tests/pauli/test_pauli_arithmetic.py

```diff
@@ -181,6 +181,9 @@ def test_operation(self, pw, op):
             assert pw_op.name == op.name
             assert pw_op.wires == op.wires
 
+        pw_tensor_op = pw.operation(get_as_tensor=True)
+        assert qml.equal(pw_tensor_op, qml.operation.Tensor(*op.operands))
```

Comment:
There was no test for checking the return value of  `pw.operation` before? I expected a change of the existing code here not adding new lines.

Reply:
yes its the same test, I just added a second assert statement instead of modifying the first.